src/libpriv/rpmostree-core.c

```diff
@@ -524,6 +524,11 @@ rpmostree_context_set_repos (RpmOstreeContext *self,
 {
   g_set_object (&self->ostreerepo, base_repo);
   g_set_object (&self->pkgcache_repo, pkgcache_repo);
+
+  self->enable_fuse =
```

Comment:
Minor/bikeshed: would `enable_rofiles` be a nicer name for this? Makes the correspondence to `RPMOSTREE_BWRAP_MUTATE_ROFILES` more obvious.